docs/pages/machine-id/introduction.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Machine ID
+description: Teleport Machine ID introduction, demo and resources.
+---
+
+# Machine ID
+
+Machine ID is a daemon that programmatically issues and renews short-lived
```

Comment:
Should we distinguish between "Machine ID" (the set of Teleport features that provides short-lived credentials for machines) and "tbot" (the daemon that issues and renews certificates)?

While `tbot` is the only way to leverage machine ID today, there may be other non-daemon modes in the near future.

Reply:
Reworded `daemon` to `service`.